azure-pipelines-review.yml

```diff
@@ -46,5 +46,19 @@ extends:
   # Django example: azure-pipelines-PROJECTNAME-api-release.yml
   # Drupal example: azure-pipelines-drupal-release.yml
   template: azure-pipelines-kultus-admin-ui-review.yml@kultus-pipelines
-  #parameters:
-    #imagetag: ${{ parameters.imagetag }}
\ No newline at end of file
+  parameters:
+    # Application build arguments and config map values as key value pairs.
+    # Does not contain all buildArguments, the rest located in kultus-pipelines/kultus-admin-ui-development.yml and kultus-admin-ui-common.yml
+    # The values here will override the values defined in the harrastukset-pipelines repository.
+    buildArgs:
+      REACT_APP_API_URI: 'https://kultus-local.api.dev.hel.ninja/graphql'
+      REACT_APP_API_REPORT_URI: 'https://kultus-local.api.dev.hel.ninja/reports'
+
+      REACT_APP_LINKEDEVENTS_API_URI: 'https://api.hel.fi/linkedevents-test/v1'
```

Comment:
Is this really available? At least it does not work with the production CMS! I think the new proper URL is `https://linkedevents.api.test.hel.ninja/v1/event/`

Reply:
done and this has changed to test pipeline too